chemicaltoolbox/openbabel/subsearch.xml

```diff
@@ -68,7 +68,7 @@ Substructure search in based on Open Babel FastSearch_ Index. It uses molecular
 
 **Input**
 
-SMILES or SMARTS patterns are possible. SD- and InChI files are converted to SMILES.
+SMILES or SMARTS patterns are possible. SDF and InChI files are converted to SMILES.
```

Comment:
They are actually called SD-files... So SDF-files is like StructuredDataFiles-files